src/txn/txn.c

```diff
@@ -2009,6 +2050,7 @@ __wt_txn_commit(WT_SESSION_IMPL *session, const char *cfg[])
 
     WT_TRET(__wt_session_reset_cursors(session, false));
     WT_TRET(__wt_txn_rollback(session, cfg));
+
```

Comment:
empty line

Reply:
Fixed in f08e9354669258aef7f1ed7607f4960fabd9ae1b